src/OpenLoco/src/Ui/Windows/VehicleList.cpp

```diff
@@ -1000,9 +1000,10 @@ namespace OpenLoco::Ui::Windows::VehicleList
         auto widgetIndex = getTabFromType(static_cast<VehicleType>(self.currentTab));
         WindowManager::invalidateWidget(WindowType::vehicleList, self.number, widgetIndex);
 
-        updateVehicleList(&self);
-        updateVehicleList(&self);
-        updateVehicleList(&self);
+        // Why is this called 3 times? Accident? What?!
+        updateVehicleList(self);
+        updateVehicleList(self);
+        updateVehicleList(self);
```

Comment:
It's quite primitive, but this is how it worked in vanilla. Each iteration only ends up updating one item, so that's why it's invoked three times. The onUpdate event is called 40 times a second, so it's an interesting solution, haha.

Reply:
I'll remove the comment.